tremor-script/src/registry.rs

```diff
@@ -853,22 +853,31 @@ impl Aggr {
         self.functions.get(module)
     }
 }
+///modifié
+// Test utility to grab a function from the registry
+pub fn fun<'event>(m: &str, f: &str) -> impl Fn(&[&Value<'event>]) -> FResult<Value<'event>> {
+    let f = registry()
+        .find(m, f)
+        .expect("could not find function")
+        .clone();
+    move |args: &[&Value]| -> FResult<Value> { f.invoke(&EventContext::new(0, None), args) }
+}
 
-#[cfg(test)]
-pub use tests::fun;
+// #[cfg(test)]
+// pub use tests::fun;
 #[cfg(test)]
 mod tests {
     use super::*;
     use simd_json::prelude::*;
 
-    // Test utility to grab a function from the registry
-    pub fn fun<'event>(m: &str, f: &str) -> impl Fn(&[&Value<'event>]) -> FResult<Value<'event>> {
-        let f = registry()
-            .find(m, f)
-            .expect("could not find function")
-            .clone();
-        move |args: &[&Value]| -> FResult<Value> { f.invoke(&EventContext::new(0, None), args) }
-    }
+    // // Test utility to grab a function from the registry
```

Comment:
If this code is not needed, let's delete it, commented out code loses its meaning quickly.

Reply:
I really don't know why there are changes there, I don't remember making them

Reply:
Oh my bad, the function was just moved above sorry :woman_facepalming:

Reply:
this seems to be fixed, can we resolve the comment?